src/main/resources/eventsPage.json

```diff
@@ -1,68 +1,78 @@
 {
-  "hero": {
-    "title": "Online and In-Person Events",
-    "description": "Join the Women Coding Community for events and meetups that enhance your skills, deepen your knowledge, and expand your professional network.\n\nOur events include webinars, hands-on workshops, study groups, panel discussions, and keynotes with industry experts. Whether online or in-person, these gatherings offer valuable opportunities to share insights, learn collaboratively, and stay updated on the latest trends. Connect, grow, and thrive with fellow members in a supportive environment.",
-    "image": {
-      "path": "https://cloudprovider.com/image.png",
-      "alt": "There is a group of women showing WCC logo",
-      "type": "DESKTOP"
-    }
-  },
-  "contact": {
-    "title": "Contact us",
-    "links": [
-      {
-        "type": "EMAIL",
-        "link": "london@womencodingcommunity.com"
-      },
-      {
-        "type": "SLACK",
-        "link": "https://join.slack.com/t/womencodingcommunity/shared_invite/zt-2hpjwpx7l-rgceYBIWp6pCiwc0hVsX8A"
+  "metadata": {
+    "pagination": {
+      "totalItems": 1,
+      "totalPages": 1,
+      "currentPage": 1,
+      "pageSize": 10
+    },
+    "hero": {
```

Comment:
In my opinion hero is not part of metadata but the page itself.

Reply:
The metadata part of the page is - that do not change, while the data part is events that will be changing we do not want the entire page to refresh just the events section so that's my thought of putting hero section in the metadata

Reply:
resordered the sections